go.mod

```diff
@@ -0,0 +1,5 @@
+module github.com/Percona-Lab/promconfig
+
+go 1.14
+
+require github.com/prometheus/common v0.9.1
```

Comment:
That dependency brings ~50 transitive dependencies 😱.

Reply:
Let's copy a part of it we need then.